# Welcome to PankBase

![image](https://github.com/user-attachments/assets/b2456a40-1eb3-4974-8a6d-211ef7bec8a4)

## Our Mission
PanKbase is a comprehensive, centralized resource for the study of the human pancreas and diabetes. The PanKbase collective aims to integrate diverse type 1 diabetes (T1D) datasets with expert-curated knowledge in a centralized, open-source data hub. Since users will ultimately be able to contribute their own data, this will be a repository for reproducible, collaborative research within the pancreas and T1D realms.

PanKbase plans to offer integrated and computation-ready genomic, epigenomic, transcriptomic, imaging, and physiological data. Datasets are currently sourced from key external programs, from which human donor-derived pancreatic data and metadata are aggregated, harmonized, and disseminated.

Read more at https://www.pankbase.org/projects.html.

## Discussion forum

We have a discussion forum set up at https://github.com/PanKbase/external-feedback/discussions. 

## Our repositories

* View our [Platform repositories](https://github.com/topics/pankbase-platform)
* View our [Pipeline repositories](https://github.com/topics/pankbase-pipeline)
* View our [PankGraph repositories](https://github.com/topics/pankbase-pankgraph)

## Platform Repositories

| Repository | Description |
|------------|-------------|
| [PanKbase UI](https://github.com/PanKbase/dig-dug-portal) | Vue‑based portal powering the main PanKbase website (pankbase.org), forked from Broad Institute’s dig‑dug‑portal. |
| [PanKbase donor summary](https://github.com/PanKbase/PanKbase-donor-summary) | R Shiny app summarizing donor metadata and clinical info. |
| [Data Library UI](https://github.com/PanKbase/igvf-ui) | Next.js‑based Data Library UI used by PanKbase, forked from IGVF‑DACC’s interface. |
| [Data Library Metadata Database](https://github.com/PanKbase/igvfd) | Metadata database for the PanKbase project, forked from IGVF‑DACC/igvfd. |
| [pankbase client openapi spec](https://github.com/PanKbase/pankbase-client-openapi-spec) | OpenAPI specification defining the REST API for the PanKbase data portal.|
| [pankbase python client](https://github.com/PanKbase/pankbase-python-client) | Auto‑generated Python client library for the PanKbase API. |
| [cdk prod](https://github.com/PanKbase/cdk-igvf-prod) | CDK infrastructure repo for deploying PanKbase Data Library services in the prod AWS account. |
| [data_analysis](https://github.com/PanKbase/data_analysis) | Jupyter notebooks and scripts for analyzing datasets within PanKbase Data Library via.  API interactions. |


## Pipeline Repositories
| Repository | Description |
|------------|-------------|
| [PanKbase scRNA seq](https://github.com/PanKbase/PanKbase-scRNA-seq) | Code and docs for replicating PanKbase’s islet single-cell RNA‑seq map. |
| [PanKbase DEG analysis](https://github.com/PanKbase/PanKbase-DEG-analysis) | Jupyter notebooks & code to replicate differential expression (DEG) analysis on PanKbase. |
| [snRNAseq NextFlow](https://github.com/PanKbase/snRNAseq-NextFlow) | A Nextflow pipeline for processing 10X single-nucleus RNA‑seq data. |
| [PanKbase RNA-seq](https://github.com/PanKbase/gtex-pipeline) | Bulk RNA‑seq pipeline to process PanKbase data forked from GTEx Consortium: alignment, quantification, QC in Docker/WDL.|
| [PanKbase snATAC-seq pipeline](https://github.com/PanKbase/HPAP-scATAC-seq) | Pipeline for single‑cell ATAC‑seq data. |

## PanKgraph Repositories
| Repository | Description |
|------------|-------------|
| [PanKgraph finemap coloc](https://github.com/PanKbase/PanKgraph-finemap-coloc) | Jupyter notebooks & code for fine-mapping and colocalization analyses in PanKbase & PanKgraph. |

(Last edited on June 16, 2025)
